src/Containers/NutritionFact/NutritionFact.tsx

```diff
@@ -71,7 +71,7 @@ export const NutritionFact:React.FC<INutritionFactProps>=({entries,editMode}):Re
      * this sets max width for the main container of the component and also sets the content for the delayed label
      */
     useLayoutEffect(()=>{
-        if(rootContainerRef.current&&containerRef.current) rootContainerRef.current.style.maxWidth=`${containerRef.current.clientWidth+EXTRA_PIXEL}px`
+        if(rootContainerRef.current&&headingContainerRef.current) rootContainerRef.current.style.maxWidth=`${headingContainerRef.current.clientWidth+EXTRA_PIXEL}px`
```

Comment:
Why do you need this XD Why not let the width be dynamic and flex correctly?

Reply:
I told you before. If I do not do this then the width of the component its the width of its largest one, which is when I explain what a daily value is at the bottom, and we do not want this